src/artm_tests/instance_test.cc

```diff
@@ -197,8 +207,10 @@ TEST(Instance, MultipleStreamsAndModels) {
   test.instance()->CreateOrReconfigureModel(m2);
 
   for (int iter = 0; iter < 100; ++iter) {
-    test.instance()->local_data_loader()->InvokeIteration(1);
-    test.instance()->local_data_loader()->WaitIdle();
+    artm::InvokeIterationArgs inv_iter_args;
+    artm::WaitIdleArgs wait_args;
+    test.instance()->local_data_loader()->InvokeIteration(inv_iter_args);
+    test.instance()->local_data_loader()->WaitIdle(wait_args);
```

Comment:
It is not important, but if you wish to same some lines there is a way to do so:
test.instance()->local_data_loader()->InvokeIteration(artm::InvokeIterationArgs());
test.instance()->local_data_loader()->WaitIdle(artm::WaitIdleArgs());